builtin/logical/transit/path_random.go

```diff
@@ -61,6 +61,10 @@ func (b *backend) pathRandomWrite(ctx context.Context, req *logical.Request, d *
 		return logical.ErrorResponse(`"bytes" cannot be less than 1`), nil
 	}
 
+	if bytes > 128*1024 {
+		return logical.ErrorResponse(`"bytes" should be less than 131072`), nil
```

Comment:
Value is still here. Just make it a const, then you can refer to it from both places.